event model - maggie - UNSURE WHERE TO PUT IM SORRY

```diff
@@ -0,0 +1,72 @@
+import { roles } from '@/types/dataModel/roles';
+import {
+  VolunteerEntity,
+  backgroundCheckStatuses,
+} from '@/types/dataModel/volunteer';
+import { Model, Schema, model, models } from 'mongoose';
+//idk if we need the above but it was from the compassion ministries? so i left it in
+
+const EventModel = new Schema(
+  {
+    ID: {
+      type: int,
+      required: true,
+      //do i need this to be unique?
+      unique: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    location: {
+      type: Location,
+      required: true,
+    },
+    startTime: {
+      type: Date,
+      required: true,
+    },
+    endTime: {
+      type: Date,
+      required: true,
+    },
+    numVolsCurr: {
+    //idk how to put this in
+      type: Volunteer.id array,
+      required: true,
+    },
+    numVolsNeeded: {
+        type: int,
+        required: true,
+      },
+    numVolsSignedUp: {
+        type: int,
+        required: true,
+    },
+    numVolsSignedIn: {
+    //idk how to put this in
+        type: Volunteer.id array,
+        required: true,
+    },
+    numVolsSignedOut: {
+    //idk how to put this in
+        type: Volunteer.id array,
+        required: true,
+    },
+    isRecurring: {
+        //idk how to put this in
+            type: Boolean,
+            required: true,
+        },
+  },
+
+);
+
+
+//this is also from the example i was following idk if i need it or not?
```

Comment:
Everything down here needs to be switched from volunteer to event. And it all depends on the types/dataModel/event file for EventEntity.f

Honestly, I don't know exactly what it's doing.

Reply:
like this?